plugin-build/src/main/kotlin/io/sentry/android/gradle/extensions/SentryPluginExtension.kt

```diff
@@ -156,6 +156,13 @@ abstract class SentryPluginExtension @Inject constructor(project: Project) {
     val autoUploadSourceContext: Property<Boolean> = objects
         .property(Boolean::class.java).convention(true)
 
+    /**
+     * Which build types should be considered for automatic source context upload.
+     * Default is "release" only.
+     */
+    val autoUploadSourceContextBuildTypes: SetProperty<String> = objects
```

Comment:
I'm not really sure if we should limit ourselves to build types, should add support for flavors too?